web/src/components/SubscribtionForm.jsx

```diff
@@ -0,0 +1,69 @@
+import { useState } from "react";
+
+function SubscriptionForm() {
+	const [subscribedUserEmail, setSubscribedUserEmail] = useState("");
+	const [successfulFetch, setSuccessfulFetch] = useState(false);
+	const [errorMessage, setErrorMessage] = useState("");
+	const url = "/api/user-subscription";
+
+	const fetchUsersSubscription = async () => {
+		try {
+			// Clear previous error messages
+			setErrorMessage("");
+			const postedEmail = await fetch(url, {
+				method: "POST",
+				headers: {
+					"Content-Type": "application/json",
+				},
+				body: JSON.stringify({ email: subscribedUserEmail.trim() }),
+			});
+			if (postedEmail.ok) {
+				setSuccessfulFetch(true);
```

Comment:
Handling the response is covered in [this ticket](https://github.com/bkarimii/slack-dashboard/issues/28)